lib/event_updater.rb

```diff
@@ -30,7 +30,7 @@ def self.update(post)
       html = (doc.try(:to_html) || html).sub(' → ', '')
 
       detail = [
-        PrettyText.excerpt(html, 50, strip_links: true, text_entities: true, keep_emoji_images: true),
+        PrettyText.excerpt(html, 1000, strip_links: true, text_entities: true, keep_emoji_images: true),
```

Comment:
That might make the popup on the calendar super huge :(

Reply:
It does make the popup large (if there is a lot of text).

I thought it was quite nice, because you basically read the post without having to leave the calendar.

I could change this though, so that the popup only takes 100 characters or 50. The excerpt here would still be 1000, but in JS I could take less for the popup.

Reply:
I guess we could try with 1000 and see how it feels.